docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -29,7 +29,7 @@ Configure the Kubernetes API Server following this template:
 
 ```
 "apiServerConfig": {
-    "--enable-admission-plugins": "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,DefaultStorageClass,ResourceQuota,PodPreset",
+    "--enable-admission-plugins": "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,DefaultStorageClass,ResourceQuota",
```

Comment:
should we now disable PodPreset or wait for Wookies to migrate their solution to svcat PodPresets?

Reply:
I already move their PodPreset to the newest one, see: https://github.com/kyma-project/kyma/pull/247/files#diff-0f6e3fa75c9732159c0aa1035562f664